Debate/Cost.lean

```diff
@@ -109,6 +111,44 @@ theorem bob_debate_cost (o : Oracle) (alice : Alice) (vera : Vera) (t : ℕ):
   · refine exp_le_of_forall_le (fun y m ↦ ?_)
     induction y; repeat simp only [Comp.cost_pure, le_refl]
 
+/-- `Nat.ceil` adds at most one -/
+lemma Nat.ceil_le_add_one {x : ℝ} (x0 : 0 ≤ x) : ⌈x⌉₊ ≤ x + 1 := by
+  rw [natCast_ceil_eq_intCast_ceil x0]
+  exact (Int.ceil_lt_add_one x).le
```

Comment:
Can you use the upstream `Nat.ceil_lt_add_one` instead of this lemma?

Reply:
Done.